transformations/transformations/utility_routines.py

```diff
@@ -64,6 +64,17 @@ def transform_subroutine(self, routine, **kwargs):
 
         routine.body = Transformer(mapper).visit(routine.body)
 
+        #Get rid of unused import and variable
+        if self.remove:
+            for imp in FindNodes(Import).visit(routine.spec):
+                if imp.module == 'yomhook':
```

Comment:
I would use `imp.module.lower() == 'yomhook'` just to be sure that this is case-insensitive.

Reply:
Added `.lower()`